docs/provider-1password-automation.md

```diff
@@ -0,0 +1,29 @@
+# TODO incomplete notes about documentation to write
+The 1Password provider supports secrets types of `Password` and `Document`.
+* The `Password` type can get data from multiple `fields` in the entry, but not the actual password field or anything else. See [creating 1Password secrets compatible with external-secrets](#creating-compatible-secrets).
+* The `Document` type can get data from the main file (not the actual password field).
```

Comment:
I find this a little confusing.  Neither type can fetch password fields?

Reply:
oh my, don't read these yet, these we're just notes to myself, i will fix up the documentation and then ask for review on it. :)

Reply:
Docs are now updated.